### PR TITLE
stolostron/backlog#21192 list prioritizers 2.4

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/placement_managed.adoc [hunk 1]
```diff
@@ -10,51 +10,58 @@ A `Placement` resource is a namespace-scoped resource that defines a rule to sel
...
+
+** `mode` is either `Exact`, `Additive`, or `""` where `""` is Additive by default.
+    
+** In `Additive` mode, any prioritizer that is not specifically provided with configuration values is enabled with its default configurations. In the current default configuration, the _Steady_ and _Balance_ prioritizers have the weight of 1, while the other prioritizers have the weight of 0. The default configurations might change in the future, which might  change the prioritization. `Additive` mode does not require that you configure all of the prioritizers. 
+
+** In `Exact` mode, any prioritizer that is not specifically provided with configuration values has the weight of zero. `Exact` mode requires that you input the full set of prioritizers that you want, but avoids behavior changes between releases.
+  
+*** `configurations` represents the configuration of prioritizers.
+    
+****`name` is the name of a prioritizer. See the following list of valid names of the prioritizers:
```
Comment (haoqing0110) [inline clusters/placement_managed.adoc]: miss blank space I think, the preview looks like below:
Screen Shot 2022-06-06 at 10 52 00

Expect to be 
- `prioritizerPolicy` represents the policy of prioritizers.
  - `mode` is either `Exact`, `Additive`, or `""` where `""` is Additive by default.
    - In `Additive` mode, ......
    - In `Exact` mode, ......
  - `configurations` represents the configuration of prioritizers.
    - `name` is the name of a prioritizer. See the following list of valid names of the prioritizers:
      - `Balance`: Balances the decisions among the clusters
      - `Steady`: Ensures that the existing decision is stabilized
      - `ResourceAllocatableCPU` & `ResourceAllocatableMemory`: sort clusters based on the allocatable
    - `weight` defines the weight of prioritizer. .......

Comment (cadawson) [reply clusters/placement_managed.adoc]: Got it. Made those updates @haoqing0110 . Thanks!

File: clusters/placement_managed.adoc [hunk 2]
```diff
@@ -10,51 +10,58 @@ A `Placement` resource is a namespace-scoped resource that defines a rule to sel
...
+
+** `mode` is either `Exact`, `Additive`, or `""` where `""` is Additive by default.
+    
+*** In `Additive` mode, any prioritizer that is not specifically provided with configuration values is enabled with its default configurations. In the current default configuration, the _Steady_ and _Balance_ prioritizers have the weight of 1, while the other prioritizers have the weight of 0. The default configurations might change in the future, which might  change the prioritization. `Additive` mode does not require that you configure all of the prioritizers. 
+
+*** In `Exact` mode, any prioritizer that is not specifically provided with configuration values has the weight of zero. `Exact` mode requires that you input the full set of prioritizers that you want, but avoids behavior changes between releases.
+  
+** `configurations` represents the configuration of prioritizers.
+    
+***`name` is the name of a prioritizer. See the following list of valid names of the prioritizers:
```
Comment (haoqing0110) [inline clusters/placement_managed.adoc]: missed a blank space before `name`, ***`name` --> *** `name`

File: clusters/placement_managed.adoc [hunk 3]
```diff
@@ -10,51 +10,58 @@ A `Placement` resource is a namespace-scoped resource that defines a rule to sel
...
+      
+**** `Balance`: Balances the decisions among the clusters
+
+****`Steady`: Ensures that the existing decision is stabilized
```
Comment (haoqing0110) [inline clusters/placement_managed.adoc]: missed a blank space before Steady, ***`Steady` --> *** `Steady`

**Review by haoqing0110 — CHANGES_REQUESTED** [review]
A bit more comments.

File: clusters/placement_managed.adoc (hunk 3 above)
Comment (cadawson) [reply clusters/placement_managed.adoc]: Good catches. Fixed. Thanks @haoqing0110 !

**Review by haoqing0110 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**